.circleci/config.yml

```diff
@@ -76,16 +76,14 @@ jobs:
           command: |-
             mkdir -p /home/node/.npm-global
```

Comment:
no need to have `.npm-global` and set `NPM_CONFIG_PREFIX` below if no `npm link`.